.github/workflows/macos_python.yml

```diff
@@ -0,0 +1,60 @@
+# These are nightly package tests for macOS
+# focus areas:
+# - initial user experience
+# - scientific python stack
+name: macOS builds nightly
+
+on:
+  # TODO: remove PR
+  pull_request:
```

Comment:
```suggestion
```

Reply:
Yep, some of these test such as building jupyter is a nightly thing. That stack takes easily 5:30hrs.